### PR TITLE
Add spldev key to RM:factory:factory-keys page

### PR DESCRIPTION
**Pull request by kprosise**
Added a minimal amount of info on spldev support. Decided against having
an admonition pointing out that there are two u-boot keys, which is
dependent on hardware features, to keep it short. Open to adding more
info if we think it is needed. There is cleanup tasks to be done
regarding word choice and grammar, but that will be in a following
PR.

Rendered page locally, and ran Vale linter. No *new* issues reported.

This addresses Jira FFTK-1597.

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

I am open to adding additional changes, kept it fairly minimal.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 8d55b9552d84974cda2b66f8d69ecf5a17f16638 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1469/docs/artifacts/html/index.html

File: source/reference-manual/factory/factory-keys.rst [hunk 1]
```diff
@@ -96,7 +103,7 @@ recommended to rotate the keys as needed. The suggestion is to rotate them each
      MODSIGN_X509 ?= "${MODSIGN_KEY_DIR}/x509_modsign.crt"
 
      #filename for U-Boot key/certificate
-     UBOOT_SIGN_KEYNAME ?= "ubootdev"
+     UBOOT_SIGN_KEYNAME ?= "ubootdev"
```
Comment (angolini) [inline source/reference-manual/factory/factory-keys.rst:106]: I'm not sure what is happening here

File: source/reference-manual/factory/factory-keys.rst [hunk 2]
```diff
@@ -56,6 +60,9 @@ configuring the variable ``OPTEE_TA_SIGN_KEY``.
 The **ubootdev** pair is a RSA 2048 key pair used for U-Boot. This is used by
 configuring the variable ``UBOOT_SPL_SIGN_KEYNAME``.
 
+The **spldev** key pair is a RSA 2048 key pair used for U-Boot proper. This is
```
Comment (angolini) [inline source/reference-manual/factory/factory-keys.rst:63]:
```suggestion
The **spldev** key pair is a RSA 2048 key pair used for U-Boot SPL. This is
```

File: source/reference-manual/factory/factory-keys.rst [hunk 3]
```diff
@@ -56,6 +60,9 @@ configuring the variable ``OPTEE_TA_SIGN_KEY``.
 The **ubootdev** pair is a RSA 2048 key pair used for U-Boot. This is used by
```
Comment (angolini) [inline source/reference-manual/factory/factory-keys.rst:60]:
```suggestion
The **ubootdev** pair is a RSA 2048 key pair used for U-Boot proper. This is used by
```

**Review by angolini — APPROVED** [review]
Only minor change. After that it's good to go!

**Comment by doanac** [comment]
Docs for e1f28c16fdd1b394be9d084b2a1020e3cedb8b61 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1477/docs/artifacts/html/index.html

**Pull request merged**